specifyweb/frontend/js_src/lib/components/TreeView/helpers.ts

```diff
@@ -141,7 +141,7 @@ export function serializeConformation(
 const throttleRate = 250;
 export const scrollIntoView = _.throttle(function scrollIntoView(
   element: HTMLElement,
-  mode: ScrollLogicalPosition = 'center'
+  mode: ScrollLogicalPosition = 'start'
```

Comment:
By making a change here, you change the behavior for every place that is using scrollIntoView().

Instead, when calling `scrollIntoView()`, just provide `start` as the second argument in the places where you need to